astroquery/cadc/core.py

```diff
@@ -428,7 +446,10 @@ def get_image_list(self, query_result, coordinates, radius):
                 '{}?{}'.format(self.data_link_url,
                                urlencode({'ID': pid_sublist}, True)))
             for service_def in datalink.bysemantics('#cutout'):
-                access_url = service_def.access_url.decode('ascii')
+                if isinstance(service_def.access_url, str):
+                    access_url = service_def.access_url
+                else:
+                    access_url = service_def.access_url.decode('ascii')
```

Comment:
this change seems unrelated?

Reply:
Indeed. It's related to a change in votable. Breaks the integration tests right now.